pkg/config/generate.go

```diff
@@ -72,6 +72,7 @@ func Generate(cs *acsapi.ContainerService, c *Config) (err error) {
 	c.Version = versionLatest
 	c.ImageConfigFormat = "openshift/origin-${component}:${version}"
 	c.TunnelHostname = strings.Replace(cs.Properties.OrchestratorProfile.OpenShiftConfig.PublicHostname, "openshift", "openshift-tunnel", 1)
+	c.EtcdPVCStorageClassName = "managed-premium"
```

Comment:
Why not set this directly in the PVC? It's always going to be managed-premium or whatever gives etcd an SSD.